src/nvim/api/win_config.c

```diff
@@ -901,7 +901,7 @@ static bool parse_bordertext_pos(String bordertext_pos, BorderTextType bordertex
   }
 
   if (bordertext_pos.size == 0) {
-    *align = kAlignLeft;
+    *align = bordertext_type == kBorderTextTitle ? fconfig->title_pos : fconfig->footer_pos;
```

Comment:
This assignment doesn't do anything. The value is `*align` is already equal to RHS.